### PR TITLE
doc/cephadm: linking to log material

### PR DESCRIPTION
**Pull request by zdover23**
This PR rewrites a section in the Troubleshooting
chapter of the Cephadm Guide. The material that this
section discusses has been covered already in the
Cephadm Guide in the Cephadm Operations chapter.
There's no reason to repeat this information twice,
unless adding technical debt to the documentation
is our goal (which of course it is not, and the
opposite of adding technical debt to the documentation
has been the aim that has guided my work these past
six months).

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by zdover23** [comment]
jenkins test docs

**Review by jdurgin — APPROVED** [review]
Good idea to dedup - as the bot shows though, the link label isn't there - looks good once that's fixed

File: doc/cephadm/troubleshooting.rst [hunk 1]
```diff
@@ -109,16 +109,11 @@ This will return something in the following form:
 Checking cephadm logs
 ---------------------
 
-You can monitor the cephadm log in real time with::
+To learn how to monitor the cephadm logs as they are generated, read :ref:`watching_cephadm_logs`.
```
Comment (neha-ojha) [inline doc/cephadm/troubleshooting.rst:112]: https://jenkins.ceph.com/job/ceph-pr-docs/50986/consoleFull#-85362248744e9240e-b50a-4693-bac0-8a991bac86ac

**Comment by zdover23** [comment]
jenkins test docs

**Pull request merged**